tests/drivers/sensor/generic/src/main.c

```diff
@@ -103,7 +103,7 @@ static struct trigger_sequence trigger_elements[] = {
  *
  * @see sensor_sample_fetch(), sensor_channel_get()
  */
-void test_sensor_get_channels(void)
+ZTEST(test_sensor_api, test_sensor_get_channels)
```

Comment:
lets drop the leading test_ from the suite name.

Reply:
Done.